Tests/MapboxMapsTests/Snapshot/MapboxMapsSnapshotTests.swift

```diff
@@ -1,10 +1,134 @@
 import XCTest
+import MapboxMaps
+import MapboxCoreMaps
+import Turf
+import Foundation
+import UIKit
 #if canImport(MapboxMaps)
 @testable import MapboxMaps
 #else
 @testable import MapboxMapsSnapshot
 #endif
-
 //swiftlint:disable explicit_top_level_acl explicit_acl
-class MapboxMapsSnapshotTests: XCTestCase {
+class MapboxMapsSnapshotTests: IntegrationTestCase {
+   
+    var geoJSONSource = GeoJSONSource()
+    var snapshotView = UIImageView()
+    var blankAttachment: XCTAttachment!
+    var newAttachment: XCTAttachment!
+    
+    // Create snapshot options
+    private func snapshotterOptions(size: CGSize) -> MapSnapshotOptions {
```

Comment:
Looks like all the tests use size 300x300. Can we just hard-code that in this method and remove the `size:` argument?